openml_OS/models/api/v1/Api_evaluation.php

```diff
@@ -40,27 +40,29 @@ private function evaluation_request($evaluation_engine_id, $order, $segs) {
     for ($i = 0; $i < count($segs); $i += 2) {
       $query_string[$segs[$i]] = urldecode($segs[$i+1]);
       if (in_array($segs[$i], $legal_filters) == false) {
-        $this->returnError(546, $this->version, $this->openmlGeneralErrorCode, 'Legal filter operators: ' . implode(',', $legal_filters) .'. Found illegal filter: ' . $segs[$i]);
+        $this->returnError(1011, $this->version, $this->openmlGeneralErrorCode, 'Legal filter operators: ' . implode(',', $legal_filters) .'. Found illegal filter: ' . $segs[$i]);
```

Comment:
Would be nice to generate an overview of error codes per API call. It becomes very hard to track like this...

Reply:
agreed